Portfolio/index.css

```diff
@@ -1,174 +1,123 @@
-console.log('Hello World');
 
-body {
-    font-family: Arial, sans-serif;
-    margin: 0;
-    padding: 0;
-    background-color: #f0f0f0;
-}
-/* Additional CSS styles */
-.exp img{
 
-width: 200px;
-height: 200px;
-border: 2px solid black;
-border-radius: 100px;
 

```

Comment:
The closing curly brace for the '.btn button' selector is missing at the beginning of line 16.